src/utils/rules-proxy.js

```diff
@@ -59,13 +59,8 @@ const getCountry = function () {
 
 const createRewriter = async function ({ distDir, projectDir, jwtSecret, jwtRoleClaim, configPath }) {
   let matcher = null
-  const configFiles = [
-    ...new Set(
-      [path.resolve(distDir, '_redirects'), path.resolve(projectDir, '_redirects'), configPath].filter(
-        (configFile) => configFile && configFile !== projectDir,
```

Comment:
It seems to me `configFile` can never equal `projectDir`. Both are returned by `@netlify/config` as `configPath` (which can be `undefined`) and `buildDir` (which is never `undefined`) respectively. The first is always a regular file, the second is always a directory.